modules/services/vicinae.nix

```
@@ -0,0 +1,184 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+let
+  cfg = config.services.vicinae;
+in
+{
+  meta.maintainers = [ lib.maintainers.leiserfg ];
+  options.services.vicinae = {
+    enable = lib.mkEnableOption "vicinae launcher daemon" // {
+      default = false;
+    };
```

```suggestion
    enable = lib.mkEnableOption "vicinae launcher daemon";
```

Default is already false.